frame/nomination-pools/src/lib.rs

```diff
@@ -2160,6 +2160,30 @@ impl<T: Config> Pallet<T> {
 		None
 	}
 
+	/// Returns the points to balance conversion for a specified pool.
+	///
+	/// If the pool ID does not exist, it returns an error. Used by runtime API.
+	pub fn api_points_to_balance(pool_id: u32) -> Result<BalanceOf<T>, ()> {
+		if let Some(pool) = BondedPool::<T>::get(pool_id) {
+			Ok(pool.points_to_balance(pool.points))
+		} else {
+			Err(())
```

Comment:
This might be my opinionated view but instead of returning a `Result`, just return a balance that is zero in the `Err` case - this will keep the return type of the API call consistent in all cases. 

 I also see this pattern being used already with `pending_rewards` whereby `0` is returned if I query an account that is not in a pool.

Reply:
Yes, that makes sense. I'll refactor both methods to return a `Balance`.